.github/workflows/production-types.yml

```
@@ -0,0 +1,84 @@
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - "main"
+    paths:
+      - "docs/docs/spec/data-types.yaml"
+jobs:
+  Push-To-Types-Repo-Staging:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    environment: production
+    env:
+      TARGET_BRANCH: "main"
+      TMP_SRCDIR: "./source"
+      TMP_DSTDIR: "./dest"
+      TMP_REPODEVDIR: "./repodev"
+      GH_TOKEN: ${{secrets.GH_TOKEN}}
+    steps:
+      - run: |
+          echo PR with changes to datatypes.yml was merged into staging
+      - name: Checkout dc-api-v2
+        uses: actions/checkout@v3
+        with:
+          path: "${{ env.TMP_SRCDIR }}"
+      - uses: actions/setup-node@v3
+        with:
+          cache-dependency-path: "${{ env.TMP_SRCDIR }}/package-lock.json"
+          node-version: 16.x
+          cache: "npm"
+      - name: Set short git commit SHA
+        working-directory: "${{ env.TMP_SRCDIR }}"
+        id: short-sha
+        run: |
+          echo "short-sha=$(git rev-parse --short ${{ github.sha }})" >> $GITHUB_OUTPUT
+      - name: get-api-version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@main
+        with:
+          path: "${{ env.TMP_SRCDIR }}"
+      - name: Generate Typescript file
+        working-directory: "${{ env.TMP_SRCDIR }}"
+        run: |
+          npm ci
+          npx openapi-typescript docs/docs/spec/data-types.yaml --output schema.ts
+      - name: Checkout nulib/dcapi-types
+        uses: actions/checkout@v3
+        with:
+          path: ${{ env.TMP_DSTDIR }}
+          ref: ${{ env.TARGET_BRANCH }}
+          repository: "nulib/dcapi-types"
+          token: ${{env.GH_TOKEN}}
+      - run: mv -f "${{ env.TMP_SRCDIR }}/schema.ts" "${{ env.TMP_DSTDIR }}/schema.ts"
+        shell: bash
+      - working-directory: "${{ env.TMP_DSTDIR }}"
+        run: |
+          git checkout -b dcapi-${{ steps.vars.outputs.short_sha }}
```

Looks like the output name defined on line 39 was `short-sha`, not `short_sha`
```suggestion
          git checkout -b dcapi-${{ steps.vars.outputs.short-sha }}
```